host.go

```diff
@@ -660,6 +723,30 @@ func (h *Host) InitCommands(c *chat.Commands) {
 				return errors.New("user not found")
 			}
 
+			oldID := member.ID()
+			newID := sanitize.Name(args[1])
+
+			// check nick registration
```

Comment:
Don't think we need this here, right? `/rename` can only be called by admins, and it should override any default behaviour anyway.

Reply:
There are two levels of authority - the users in the admin list (operators), including those granted ops by other ops, and the folks who have write privileges to the admin and allowlist files. I thought it more trustworthy to limit the class of people who can override the nick reservations, to maximize the amount trust users could put in the system.